cmd/root.go

```diff
@@ -154,9 +155,9 @@ func initConfig() {
 		if err != nil {
 			fmt.Print("unable to find home directory")
 		}
-		// Search config in home directory with name ".generic" (without extension).
+		// Search config in home directory with name ".epr" (without extension).
 		viper.AddConfigPath(home)
```

Comment:
Not pertinent to PR - should this use xdg config home instead of user home?

Reply:
Yes we should. #31